consensus/protocol.go

```diff
@@ -74,4 +74,7 @@ type Peer interface {
 
 	// SendConsensus sends the message to this p2p peer using the consensus specific devp2p subprotocol
 	SendConsensus(msgcode uint64, data interface{}) error
+
+	// SendQbftConsensus is used to send consensus subprotocol messages from an "eth" peer without encoding the payload
+	SendQbftConsensus(msgcode uint64, payload []byte) error
```

Comment:
Should we uppercase `Qbft`: `SendQbftConsensus` -> `SendQBFTConsensus`

Reply:
The idea here was to have function names in camelcase so they are more readable. I can update the names if you think `SendQBFTConsensus` would be more readable